DQM/BeamMonitor/test/beamspotdip_dqm_sourceclient-file_cfg.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import print_function
+import FWCore.ParameterSet.Config as cms
+
+#
+process = cms.Process("BeamSpotDipServer")
+process.load("DQMServices.Core.DQM_cfg")
+
+# message logger
+process.load("FWCore.MessageLogger.MessageLogger_cfi")
+process.MessageLogger = cms.Service("MessageLogger",
+  debugModules = cms.untracked.vstring('*'),
+  cerr = cms.untracked.PSet(
+    FwkReport = cms.untracked.PSet(
+      optionalPSet = cms.untracked.bool(True),
+      reportEvery = cms.untracked.int32(999999999),
+    )
+  ),
+  destinations = cms.untracked.vstring('cerr'),
+)
+
+# source
+process.source = cms.Source("PoolSource",
+  fileNames=cms.untracked.vstring(
+    'file:/tmp/sikler/b.root' # lxplus7101
+  )
+)
+
+process.maxEvents = cms.untracked.PSet(
+  input = cms.untracked.int32(100)
+)
+
+# beamspot from database
+process.load("CondCore.CondDB.CondDB_cfi") 
+
+process.BeamSpotDBSource = cms.ESSource("PoolDBESSource",
+  process.CondDB,
+  toGet = cms.VPSet(
+    cms.PSet(
+      record = cms.string('BeamSpotOnlineLegacyObjectsRcd'),
+      tag = cms.string("BeamSpotOnlineTestLegacy"),
+      refreshTime = cms.uint64(1)
+    ),
+    cms.PSet(
+      record = cms.string('BeamSpotOnlineHLTObjectsRcd'),
+      tag = cms.string("BeamSpotOnlineTestHLT"),
+      refreshTime = cms.uint64(1)
+    )
+  )
+)
+
+process.BeamSpotESProducer = cms.ESProducer("OnlineBeamSpotESProducer")
+process.BeamSpotDBSource.connect = cms.string('frontier://FrontierProd/CMS_CONDITIONS')
```

Comment:
In assignment, explicit types on the right hand side should be avoided
http://cms-sw.github.io/cms_coding_rules.html#6--packaging-rules-1